### ISSUE TITLE
state expansion

### ISSUE THREAD

**Issue by yodrew** [liberland/laws#16]
Hi, Kacper,

I watched your youtube talk about the constitution and wished to make two comments. You have made a bold and valiant effort in writing Liberland's constitution, and I congratulate you on your Herculean accomplishment.

Unfortunately, you have made two fundamental errors in your understanding about the US constitution. As an American observer of America, a child of militant patriots, and a lifelong student of philosophy, including some law, I will point them out.

First, the expansion of the federal government has not come from a lack of prohibitions or pages in the constitution. It has come from a few small phrases that were used to expand the state. For example,
"...to provide for the general welfare"
was used to create the national bank and Roosevelt's innumerable new agencies, including the welfare state. In fact, the ratifying states objected to this phrase in anticipation of state expansion. Hamilton and the Federalists reassured them the phrase would never be used this way. Then it was used this way beginning shortly thereafter. 

For more about this, I recommend Tom Woods debate with Michael Malice about Alexander Hamilton. Tom Woods is extremely knowledgeable about this subject and might be willing to consult on the constitution.

Second, your constitution's many negative statements on what the state may not do contradict the purpose of a constitution and will lead to precisely what you fear. The point of a constitution is to describe what a state may do. If it is not specifically allowed, it is forbidden. If you write what the state may _not_ do, you imply the state has undescribed powers. Again, US governments at every level have used this to gradually expand their power. 

The Bill of Rights, for example, was a massive error. Firstly, the natural rights of individuals and state powers in relation to them were well understood and codified in the English common law. The constitution officially recognizes this law as its basis. But even if codifying them were a good idea, their negative context as regards state powers poisons them. It gave the state a chance to define individual rights in increasingly narrow ways. 

The 2nd Amendment is a famous example. Gun controllers have often succeeded in legislating that people should only have guns if they are in well-regulated militias, such as the state-operated National Guard.

In summary, you are too guarded in the text against state expansion. You have multiplied laws unnecessarily. As the state of Liberland has fewer powers than the US state, its constitution ought to be shorter. I suggest that you aim for one paragraph. Or one page. Then you will end up with a five page document that is much easier to read and simpler to apply for everyone. The longer it is, the more avenues of attack you give lawyers and legislators to twist the state to evil ends.

I hope you will consider these matters further and correct the constitution accordingly. 

Sincerely,
Andrew Durham

**Comment by michalptacnik** [comment]
Dear Andrew, 
We have completely reviewed the Constitution and removed many of the "negative delimitations" you pointed out in your post. You can see our draft constitution here: https://docs.google.com/document/d/1RYgEHcb2oMgYJOa2MWUxe8E0aHRIgDpsiMG21MACIVg/edit#heading=h.yur44cl44s61 and we would be happy if you added your comments to it.
Closing the issue with this, please reopen if you feel some of the comments need to be addressed here rather than in the constitution document.
